benchopt/cli/main.py

```diff
@@ -28,14 +59,14 @@
 )
 @click.argument('benchmark', type=click.Path(exists=True),
                 shell_complete=complete_benchmarks)
-@click.option('--objective-filter', '-o', 'objective_filters',
+@click.option('--objective-filter', '-o', OPT_TO_VAR['--objective-filter'],
               metavar='<objective_filter>', multiple=True, type=str,
               help="Filter the objective based on its parametrized name. This "
               "can be used to only include one set of parameters.")
 @click.option('--old_objective-filter', '-p', 'old_objective_filters',
               multiple=True, type=str,
               help="Deprecated alias for --objective_filters/-o.")
-@click.option('--solver', '-s', 'solver_names',
+@click.option('--solver', '-s', OPT_TO_VAR['--solver'],
```

Comment:
an alternative is to remove all the variable names in options, let click do the conversion automatically and convert back when needed in `_get_run_args`. this way no need for `OPT_TO_VAR`

We keep to chose the variable names when we declare all the variables that unpack the result of `_get_run_args`